doc/src/manual/strings.md

```diff
@@ -883,3 +883,11 @@ scheme.
 
 Besides being used for the [`VERSION`](@ref) constant, `VersionNumber` objects are widely used
 in the `Pkg` module, to specify packages versions and their dependencies.
+
+## [Raw String Literals](@id man-raw-string-literals)
+
+Raw strings without interpolation or unescaping can be expressed with
+non-standard string literals of the form `raw"..."`. Raw string literals
+create ordinary `String` objects which contain the enclosed contents exactly
+as entered with no interpolation or unescaping. This is useful for strings which
```

Comment:
won't get addressed if it remains a comment on a closed pr. open an issue or pr?